Resources/config/services/services.yml

```diff
@@ -2,19 +2,6 @@ parameters:
   tactician.commandbus.class: League\Tactician\CommandBus
 
 services:
-    tactician.handler.locator.symfony:
```

Comment:
These services are now aliases to the default locator/handler created in the compiler pass